contrib/guix/INSTALL.md

```diff
@@ -0,0 +1,679 @@
+   - Works on most Linux distributions
+   - Installs any release
+   - Binary installation only, requires high level of trust
+4. Using a **distribution-maintained package** [⤓ skip to section]()
+   - Maintained by distribution's Guix package maintainer
+   - Normal difficulty (manual setup required)
+   - Works only on distributions with Guix packaged, at time of writing:
+     - Debian Bullseye or later
+     - Ubuntu Hirsute or later
+     - Arch Linux (via AUR)
+   - Installs a release decided on by package maintainer
+   - Source or binary installation depending on distribution
+5. Building **from source** [⤓ skip to section]()
+   - Maintained by you
+   - Hard, but rewarding
+   - Can be made to work on most Linux distributions
+   - Installs any commit (more granular)
+   - Source installation, requires lower level of trust
+
+## Options 1 and 2: Using the official shell installer script or binary tarball
+
+The instructions for both using the offical shell installer script and using
+binary tarballs are both found in the GNU Guix Manual's [Binary Installation
+section](https://guix.gnu.org/manual/en/html_node/Binary-Installation.html).
+
+Note that running through the binary tarball installation steps is largely
+equivalent to manually performing what the shell installer script does.
+
+Note that at the time of writing (July 5th, 2021), the shell installer script
+automatically creates an `/etc/profile.d` entry which the binary tarball
+installation instructions do not ask you to create. However, you will likely
+need this entry for better desktop integration. Please see [this section]() for
+instructions on how to add a `/etc/profile.d/guix.sh` entry.
+
+Regardless of which installation option you chose, the changes to
+`/etc/profile.d` will not take effect until the next shell or desktop session,
+so you should log out and log back in.
+
+## Option 3: Using fanquake's Docker image
+
+Please refer to fanquake's instructions
+[here](https://github.com/fanquake/core-review/tree/master/guix).
+
+Note that the `Dockerfile` is largely equivalent to running through the binary
+tarball installation steps.
+
+## Option 4: Using a distribution-maintained package
+
+### Debian 11 ("Bullseye")/Ubuntu 21.04 ("Hirsute")
+
+Guix v1.2.0 is available as a distribution package starting in [Debian
+11](https://packages.debian.org/bullseye/guix) ("bullseye") and [Ubuntu
+21.04](https://packages.ubuntu.com/hirsute/guix) (Hirsute Hippo).
+
+Note that if you intend on using Guix without using any substitutes (more
+details [here]()), v1.2.0 has a known problems when building GnuTLS from source.
+Solutions and workarounds are documented [here]().
+
+
+To install:
+```sh
+sudo apt install guix
+```
+
+For up-to-date information on Debian and Ubuntu's release history:
+- [Debian release history](https://www.debian.org/releases/)
+- [Ubuntu release history](https://ubuntu.com/about/release-cycle)
+
+### Arch Linux
```

Comment:
no opinion, but we don't provide Arch Linux specific instructions anywhere else

Reply:
I think we do here: https://github.com/bitcoin/bitcoin/blob/master/doc/build-unix.md#setup-and-build-example-arch-linux